pytorch_lightning/utilities/__init__.py

```diff
@@ -50,6 +53,7 @@ def _module_available(module_path: str) -> bool:
 
 TPU_AVAILABLE = XLADeviceUtils.tpu_device_exists()
 FAIRSCALE_AVAILABLE = platform.system() != 'Windows' and _module_available('fairscale.nn.data_parallel')
+BOLT_AVAILABLE = _module_available('pl_bolts')
```

Comment:
Do we really need this here? Bolts should depend on PL, but there shouldn't be any code in PL that depends on bolts...

Reply:
In the new example, I am using bolt for download cifar10.